webapp/src/main/webapp/angular.json

```diff
@@ -80,9 +80,7 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "microcks-ui:build",
-            "proxyConfig": "proxy.conf.json"
-          },
+            "browserTarget": "microcks-ui:build"          },
```

Comment:
`proxy.conf.json` is necessary while in development mode. I use `ng serve` to serve the frontend with hot-reload and backend is ran with `mvn spring-boot:run`

Reply:
Actually i do the same with `ng serve` and `mvn spring-boot:run` but I changed that the Angular services do not use the hardcoded `/api/` path instead the (https://github.com/fhopfensperger/microcks/blob/4e3abe87d93030588c2ebc6bd50f115db4a5282c/webapp/src/main/webapp/src/app/services/importers.service.ts#L27) services load the actual address + path from the ` environment`. Because of this I have configured the `environment.ts` file and removed the not longer needed `proxy.conf.json`:
```
export const environment = {
  production: false,
  apiUrl: 'http://localhost:8080/',
  baseContext: '/'
};
```

Reply:
But something just comes to my mind, we could define in the environment only the `/` as `apiUrl` and keep the proxy.conf. Maybe we can fix the CORS issue as well. If you want I can try it out.